gef.py

```diff
@@ -2230,6 +2235,23 @@ def get_info_sections():
 
     return
 
+def get_saved_ip():
+    """Retrieves the saved ip using the 'info frame' command ."""
+
+    lines = gdb.execute("info frame", to_string=True).splitlines()
+    if lines:
+        saved_regs = lines[len(lines)-1].split(" ")
```

Comment:
* `lines[-1]` to get the last line.
* `split()` on its own splits on spaces, no need for an arg.